egui/src/widgets/plot/mod.rs

```diff
@@ -208,11 +213,43 @@ impl Plot {
     /// .show(ui, |plot_ui| plot_ui.line(line));
     /// # });
     /// ```
-    pub fn custom_label_func<F: 'static + Fn(&str, &Value) -> String>(
+    pub fn hover_label_func<F: 'static + Fn(&HoverConfig, &str, &Value) -> String>(
         mut self,
-        custom_lebel_func: F,
+        hover_label_func: F,
     ) -> Self {
-        self.custom_label_func = Some(Box::new(custom_lebel_func));
+        self.hover_label_func = Box::new(hover_label_func);
+        self
+    }
+
+    pub fn default_hover_label_func() -> Box<dyn Fn(&HoverConfig, &str, &Value) -> String> {
```

Comment:
Why is this public? Someone who wants the default formatter would simply not call `hover_label_func()`.

Reply:
That's a good point. Having it public would have made the `plot_demo` code slightly easier to write, but that's probably too much of an edge case.